### DISCUSSION TITLE
Using Content v2 with external Markdown Sources

### DISCUSSION THREAD

**Discussion by nevotheless** [nuxt/nuxt#17555] [Questions] [answered]
Howdy 👋🏽 

This is more about Content v2... but since it's kinda public right now (even if not fully released), i gave it a shot and tried rendering the markdown i'm getting via Strapi with it. But to no success. 

I'm having a hard time wrapping my head around the docs and example (which are seemingly only meant to be consumed for the "filesystem cms"?). 

But i'm not very experienced with vue and nuxt so maybe i'm missing something obvious here. But i'm certain that it's the latter since i cannot imagine that it wouldn't be possible to render markdown from external sources right? 🤞🏽 

I saw a content v1 example where i could've just used something like 
```js
return new Markdown(options.markdown).toJSON(md)
```
where md could be my markdown string. Is there something similar in content v2 and if yes, i would appreciate if you could point me into the right direction. 😃

**Comment by benoitdemaegdt** [comment, marked as answer]
Hey @nevotheless 

Rendering markdown from other providers such as Notion or Strapi is still an **experimental feature** of Nuxt Content V2. So far, there is no built-in way to format Strapi (or Notion) data to a Nuxt Content readable format. You may have to write custom code to do this 🤔 .

Here is an issue from one of the main Nuxt Content V2 contributor talking about this feature : https://github.com/nuxt/content/issues/1049
